test/Benchmarks/README.md

```diff
@@ -3,5 +3,5 @@
 Use the following example to run Benchmarks from command line:
 (change parameters as necessary)
 
-1. navigate to opentelemetry-dotnet\src\benchmarks directory and run the following
+1. navigate to opentelemetry-dotnet\test\benchmarks directory and run the following
```

Comment:
```suggestion
Navigate to `./test/benchmarks` directory and run the following:
```

Reply:
Use `/` in all the docs for consistency.

Reply:
just updated!